images/ose-cluster-kube-descheduler-operator.yml

```diff
@@ -1,4 +1,3 @@
-# [asdas] Disabling until upstream adds 4.15 configs to stable dir
 mode: disabled
```

Comment:
maybe remove this line

Reply:
apologies, I removed the comment but not the mode. Fixed